### ISSUE TITLE
Parts of traces disappearing on the website

### ISSUE THREAD

**Issue by Zverik** [openstreetview/openstreetview.org#36]
Firefox 47, Linux
See http://openstreetview.org/map/@57.959636224196,27.596712112426758,13z
Scroll in little steps to the north. Segment at the bottom start disappearing, and new segments abruptly appear at the top.

**Comment by james2432** [comment]
Duplicate of #1

**Comment by Zverik** [comment]
No, this issue is about a web interface glitch, not extra ways shown.

**Comment by alexandrui-telenav** [comment]
hi @Zverik , I can reproduce this on Safari as well so it's a general problem in the display logic.  I added this to the backlog to be fixed.
Thanks for reporting, I'll give you an update when it's fixed.
